examples/hello_alt_methods/src/tests.rs

```diff
@@ -0,0 +1,31 @@
+use super::rocket;
+use rocket::testing::MockRequest;
+use rocket::http::Method;
+use rocket::http::Status;
+
+#[test]
+fn hello_world_alt_methods() {
+    test(Method::Get, Status::Ok, Some("<!DOCTYPE html>"));
+    test(Method::Put, Status::Ok, Some("Hello, PUT request!"));
+    test(Method::Post, Status::NotFound, None);
+}
+
+fn test(
```

Comment:
These are really long parameter names. Can we make them shorter while keeping them fairly descriptive? Maybe just `status` and `body_prefix`?